loopout.go

```diff
@@ -1331,6 +1550,59 @@ func (s *loopOutSwap) sweep(ctx context.Context,
 		}
 	}
 
+	return nil
+}
+
+// sweep tries to sweep the given htlc to a destination address. It takes into
+// account the max miner fee and marks the preimage as revealed when it
+// published the tx. If the preimage has not yet been revealed, and the time
+// during which we can safely reveal it has passed, the swap will be marked
+// as failed, and the function will return.
+func (s *loopOutSwap) sweep(ctx context.Context, htlcOutpoint wire.OutPoint,
+	htlcValue btcutil.Amount) error {
+
+	confTarget, canSweep := s.sweepConfTarget()
+	if !canSweep {
```

Comment:
The flow is a bit hard to understand but in the normal sweep case (no musig), `sweepConfTarget` will mark the swap as failed if the deadline is exceeded and the preimage is not revealed and we'll at that point fail the swap. In the musig case this function is called if the coop signing fails and at that point we must we go on since the preimage is revealed so we'll try to sweep until it succeeds.